devito/logger.py

```diff
@@ -34,7 +35,8 @@
 logging.addLevelName(DSE, "DLE")
 logging.addLevelName(DSE_WARN, "DLE_WARN")
 
-logger.setLevel(INFO)
+init_log_level = eval(parameters["log_level"])
```

Comment:
Why do we need `eval` here? Shouldn't the `parameters` dictionary be initiated and accessible before we even get here? Or am I missing something here?